test/Faker/Provider/HtmlLoremTest.php

```diff
@@ -20,10 +20,28 @@ public function testProvider()
     public function testRandomHtmlReturnsValidHTMLString(){
         $faker = new Generator();
         $faker->addProvider(new HtmlLorem($faker));
-        $node = $faker->randomHtml(6, 10);
+        $node = $faker->randomHtml(5, 5);
+        $dom = new \DOMDocument();
+        $error = $dom->loadHTML($node);
```

Comment:
I think `$isValidHTML`  would make more sense than `$error` as a variable name here and below.